tests/installation/bootloader_settings/disable_boot_menu_timeout_navigating_tabs.pm

```diff
@@ -0,0 +1,18 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Disable grub timeout from the Installer according bsc#1208266
```

Comment:
instead of `according` perhaps would be more clear to phrase it like `to catch regression of`